photon-core/src/main/java/org/photonvision/PhotonVersion.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * MIT License
```

Comment:
Looks like this file was committed by accident? Please delete or I'm happy to delete it for you if I have write access.

Reply:
I would like it if you did delete them, as I am unsure how. How can I give you written access?

Reply:
Looks like he already has write access "Maintainers are allowed to edit this pull request."

Reply:
After flailing around for awhile, I came to that conclusion too

Reply:
I'm looking at it now.

Reply:
I deleted the 4 files that were added by accident and that seems to have fixed the problems.